Lib/sqlite3/test/userfunctions.py

```diff
@@ -351,6 +294,42 @@ def test_empty_blob(self):
         cur = self.con.execute("select isblob(x'')")
         self.assertTrue(cur.fetchone()[0])
 
+    def test_nan_float(self):
+        cur = self.con.execute("select boomerang(?)", (float('nan'),))
+        # SQLite has no concept of nan; it is converted to NULL
+        self.assertIsNone(cur.fetchone()[0])
+
+    def test_too_large_int(self):
+        err = "Python int too large to convert to SQLite INTEGER"
+        self.assertRaisesRegex(OverflowError, err, self.con.execute,
+                               "select boomerang(?)", (1 << 65,))
```

Comment:
There are several Python-to-SQLite-to-Python conversions here. 1) converting the binded parameters, 2) converting  the result of the user function. It is not clear on what step the exception was raised.

You need to eliminate one of steps. Since we test here binding of parameters, we should use a  function which returns a safe value, which does not cause conversion error.